src/devices/vdml_rotation.cpp

```diff
@@ -77,9 +77,9 @@ std::ostream& operator<<(std::ostream& os, const pros::Rotation& rotation) {
 
 namespace literals {
 const pros::Rotation operator"" _rot(const unsigned long long int r) {
```

Comment:
Now that we support making rotation sensors with negative ports, should this be changed to signed? Probably outside the scope of this pr

Reply:
yes